### PR TITLE
[13.0][IMP] mrp_unbuild_cust_alu_analytics_maintenance: Add maintenance_equipment_id to tree view and change translation

### PR DESCRIPTION
**Pull request by Anxo82**
Add maintenance_equipment_id to tree view and change translation

### CODE REVIEW

**Comment by Anxo82** [comment]
@ChristianSantamaria @dalonsod Review please.
Thanks

File: mrp_unbuild_cust_alu_analytics_maintenance/views/mrp_unbuild_incidence_views.xml [hunk 1]
```diff
@@ -15,4 +15,18 @@
...
+    <record id="mrp_unbuild_incidence_tree_inherit_view" model="ir.ui.view">
+        <field name="name">mrp.unbuild.incidence.tree.inherit</field>
+        <field name="model">mrp.unbuild.incidence</field>
+        <field name="inherit_id" ref="mrp_unbuild_cust_alu_analytics.view_mrp_incidence_tree"/>
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='machine']" position="attributes">
+                <attribute name="invisible">1</attribute>
+            </xpath>
+            <xpath expr="//field[@name='machine']" position="after">
+                <field name="maintenance_equipment_id"/>
```
Comment (dalonsod) [inline mrp_unbuild_cust_alu_analytics_maintenance/views/mrp_unbuild_incidence_views.xml:38]: Make it optional and shown by default

File: mrp_unbuild_cust_alu_analytics_maintenance/views/mrp_unbuild_incidence_views.xml [hunk 2]
```diff
@@ -15,4 +15,18 @@
         </field>
     </record>
 
+    <record id="mrp_unbuild_incidence_tree_inherit_view" model="ir.ui.view">
```
Comment (dalonsod) [inline mrp_unbuild_cust_alu_analytics_maintenance/views/mrp_unbuild_incidence_views.xml:18]: As added to tree view, I should add it to `search` view as well (searching for this file, and grouping too).

**Review by dalonsod — COMMENTED** [review]
See comments.

**Comment by Anxo82** [comment]
@ChristianSantamaria @dalonsod Review please.
Thanks

File: mrp_unbuild_cust_alu_analytics_maintenance/views/mrp_unbuild_incidence_views.xml [hunk 3]
```diff
@@ -15,4 +15,35 @@
...
+    <record id="mrp_unbuild_incidence_search_inherit_view" model="ir.ui.view">
+        <field name="name">mrp.unbuild.incidence.search.inherit</field>
+        <field name="model">mrp.unbuild.incidence</field>
+        <field name="inherit_id" ref="mrp_unbuild_cust_alu_analytics.view_mrp_incidence_search"/>
+        <field name="arch" type="xml">
+            <xpath expr="//search/field[@name='detail']" position="after">
+                <field name="maintenance_equipment_id"/>
+            </xpath>
+            <xpath expr="//filter[@name='unbuild_date']" position="before">
+                <filter string="Machine" name="machine_filter"  domain="[('maintenance_equipment_id', '!=', False)]"/>
```
Comment (dalonsod) [inline mrp_unbuild_cust_alu_analytics_maintenance/views/mrp_unbuild_incidence_views.xml]: Proper label:

```suggestion
                <filter string="With Machine" name="machine_filter"  domain="[('maintenance_equipment_id', '!=', False)]"/>
```

File: mrp_unbuild_cust_alu_analytics_maintenance/i18n/es.po [hunk 4]
```diff
@@ -6,61 +6,59 @@ msgid ""
...
+#: model_terms:ir.ui.view,arch_db:mrp_unbuild_cust_alu_analytics_maintenance.mrp_unbuild_incidence_search_inherit_view
+msgid "Machine"
+msgstr "Maquina"
+
 #. module: mrp_unbuild_cust_alu_analytics_maintenance
 #: model:ir.model,name:mrp_unbuild_cust_alu_analytics_maintenance.model_maintenance_equipment
 #: model:ir.model.fields,field_description:mrp_unbuild_cust_alu_analytics_maintenance.field_mrp_unbuild_incidence__maintenance_equipment_id
 msgid "Maintenance Equipment"
-msgstr "Equipo de mantenimiento"
+msgstr "Maquina"
```
Comment (dalonsod) [inline mrp_unbuild_cust_alu_analytics_maintenance/i18n/es.po]: Missing accent:

```suggestion
msgstr "Máquina"
```

**Review by dalonsod — CHANGES_REQUESTED** [review]
See minor comments. When attended please @ChristianSantamaria update AluTest environment and I'll test it

**Comment by dalonsod** [comment]
One very last thing @Anxo82 , manifest version isn't updated!

**Comment by Anxo82** [comment]
Done! Thanks

**Review by dalonsod — APPROVED** [review]
Let's go!!!

**Pull request merged**